lib/desired-caps.js

```diff
@@ -0,0 +1,102 @@
+import log from './logger';
+import validator from 'validate.js';
+
+validator.validators.isString = function(value) {
```

Comment:
gotcha. That will work because when drivers which extend BaseDriver create their own constraints objects, they don't need a reference to these functions, just use the name as a key. neat.